third_party/jemalloc.BUILD

```diff
@@ -89,6 +89,13 @@ cc_library(
         "-D_REENTRANT",
     ],
     includes = ["include"],
+    linkopts = select({
+        "@%ws%//tensorflow:linux_ppc64le": [
+            "-lpthread",
+        ],
+        "//conditions:default": [
+        ],
+    }),
```

Comment:
This change is needed to fix bazel tests in tensorflow/compiler when jemalloc is enabled. Jemalloc code uses pthread_atfork function in jemalloc.c which gets executed in case Power but not on x86. To resolve this function, we need jemalloc to be built linking libpthread.